pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -34,67 +39,71 @@ const (
 	pod       = "helloworld-go-00001-deployment-8ff587cc9-7g9gc"
 )
 
+var ignoreStatFields = cmpopts.IgnoreFields(metrics.Stat{}, "ProcessUptime")
+
 var testCases = []struct {
-	name                        string
-	reportingPeriod             time.Duration
-	concurrency                 float64
-	proxiedConcurrency          float64
-	reqCount                    float64
-	proxiedReqCount             float64
-	expectedReqCount            float64
-	expectedProxiedRequestCount float64
-	expectedConcurrency         float64
-	expectedProxiedConcurrency  float64
+	name            string
+	reportingPeriod time.Duration
+	report          network.RequestStatsReport
+	want            metrics.Stat
 }{{
 	name:            "no proxy requests",
 	reportingPeriod: 1 * time.Second,
-
-	reqCount:    39,
-	concurrency: 3,
-
-	expectedReqCount:            39,
-	expectedConcurrency:         3,
-	expectedProxiedRequestCount: 0,
-	expectedProxiedConcurrency:  0,
+	report: network.RequestStatsReport{
+		AverageConcurrency: 3,
+		RequestCount:       39,
+	},
+	want: metrics.Stat{
+		RequestCount:              39,
+		AverageConcurrentRequests: 3,
```

Comment:
Can we sort them the same way (probably alpha-sort)?

Reply:
@vagababov maybe we should refactor the [struct definition itself](https://github.com/knative/serving/blob/fccf86bc54384c7100755826ee551afe99d66351/pkg/autoscaler/metrics/stat.pb.go#L43-L56).

Reply:
It simplifies readability, when items within a block are sorted.
I can deal with it separately.

Reply:
I can do it as well here (already fixed that part), but from what I see not all code bases follow that. Readability improvements should apply also in places outside tests like struct definition and wherever that struct is used and more.

Reply:
Sure — it's one of those: see do things.